spec/support/fake_sms.rb

```diff
@@ -1,5 +1,5 @@
 class FakeSms
-  Message = Struct.new(:from, :to, :body)
+  Message = Struct.new(:from, :to, :body, :messaging_service_sid)
```

Comment:
should we remove `from` if we're no longer using it?

Reply:
I'll look to see if we use this for voice calls. If not then yeah, I'll remove it.